ccm_web/server/src/user/user.model.ts

```diff
@@ -18,6 +18,7 @@ interface UserCreationAttributes extends Optional<UserAttributes, 'id'> {
 
 // Sequelize Datatypes mapping with Mysql https://sequelize.org/master/manual/model-basics.html#data-types
 @Table({
+  tableName: 'user',
```

Comment:
I agree this is better way to do it. But @chrisrrowland should validate it and approve

Reply:
This works for me.

Reply:
Should put the properties in alphabetical order.


Hahahah just kidding.  I don't care.